chemdash.py

```diff
@@ -368,13 +354,45 @@ def get_dist_plot(selected_columns, data):
 
         return select_cols, fig
 
+    return app
+
+
+def main(
+    input_file: Path,
+    port: int = 8000,
+    debug: bool = False
+):
```

Comment:
This wraps calling the things at the top-level of the module.  All to change w/ django/gunicorn, but great for debugging.